english/light-fire/run-away/nothing/nothing.md

```diff
@@ -1 +1,4 @@
-The child runs across the field and grabs handfuls of marshmallow fluff from the walls of the gelatinous glob that was previously your home and starts putting them into his mouth.
\ No newline at end of file
+The child runs across the field and grabs handfuls of marshmallow fluff from the walls of the gelatinous glob that was previously your home and starts putting them into his mouth.
```

Comment:
It looks like this long line was already there, so I'll go ahead and merge this. (Casey is talking about the first line in the file. I'm not sure why it is displaying strangely on GitHub.)